### ISSUE TITLE
● Symbol in CalenderDateHoursMinutesSecondsRegexp

### ISSUE THREAD

**Issue by YoamFarges** [readefries/RFISO8601DateTime#1]
Hi, first of all, thanks for the library, it's handy!

What is the purpose of the black dot (●) in the regex? Is this a typo? It makes some valid dates returning nil because of that.

For example a dateString like `"2016-10-18 11:58:00"` would not be recognized by the _parseDateString_ method, even if this comment says otherwise: ` // Calendar date with hours, minutes, and seconds (e.g., 2008-08-30 17:21:59 or 20080830 172159).`

Cheers!

**Comment by readefries** [comment]
Hi, good to hear it's handy 😊

I'll take a look at this issue tomorrow.

**Comment by YoamFarges** [comment]
Actually I stopped using your library as I encountered some crashes with it.

Sometimes, the verification regex was passing, but the returned NSDate was nil, and thus the crash due to explicitly unwrapping optional dates everytime.
I do not remember if my diagnostic was that it was an iOS defect or not, however I ended just using several NSDateFormatter in a not-really satisfying way.

Sorry about not investigating further and fixing the library or something though, I'm a little bit busy at the moment.
